src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -15,4 +19,15 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar"
+      Link="jars\%(Filename)%(Extension)"
```

Comment:
(nit) I wasn't clear about the formatting, but we typically align the attributes underneath the attribute on the first line. This isn't 100% necessary to fix, but if you are changing something else, consider this.

Reply:
no worries, fixed